functional-tests/build.gradle

```diff
@@ -56,6 +56,9 @@ dependencies {
   }
   testCompile 'org.slf4j:slf4j-api:1.7.13'
   testCompile 'org.slf4j:slf4j-simple:1.7.13'
+
+  testCompile 'org.codehaus.groovy:groovy-all:2.3.7'
+  testCompile group: 'org.mongodb', name: 'mongo-java-driver', version: '3.6.3'
```

Comment:
Nit.: should we keep the import style the same?
`testCompile 'org.mongodb:mongo-java-driver:3.6.3'` should work, and it retains the style used throughout the rest of the code.

Reply:
Updated.